images/primary_node/repo_url_insert_script.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# -*- coding: utf-8 -*-
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script helps to insert correct repo URLs for the MapR.
+
+MAPR_VERSION="$1"
+MEP_VERSION="$2"
+
+if [ "$MAPR_VERSION" = "" ]; then
+    echo "Usage: $0 MAPR_VERSION ..."
+    exit 1
+fi 
+if [[ "$MAPR_VERSION" != "5.2.0" && "$MEP_VERSION" == "" ]]; then
+    echo "Usage: $0 MAPR_VERSION MEP_VERSION ..."
+    exit 1
+fi
+
+echo "baseurl = http://package.mapr.com/releases/v${MAPR_VERSION}/redhat/" >> /etc/yum.repos.d/mapr_core.repo
+if [[ $MAPR_VERSION = "5.2.0" && $MEP_VERSION == "" ]]; then
```

Comment:
Replace `[[` and `]]` with `[` and `]` throughout and use the POSIX-compliant forms of the statements.

Reply:
Also, use [ `-z "${MEP_VERSION}" ] instead of comparing a variable to an empty string.